test/sdk/go/sdk-client-test.go

```diff
@@ -106,6 +106,37 @@ func main() {
 		if c != capacity {
 			log.Fatalf("Player Capacity should be %d, but is %d", capacity, c)
 		}
+
+		playerID := "1234"
+		if ok, err := sdk.Alpha().PlayerConnect(playerID); err != nil {
+			log.Fatalf("Error registering player as connected: %s", err)
+		} else if !ok {
+			log.Fatalf("PlayerConnect returned false")
+		}
+
+		if ok, err := sdk.Alpha().IsPlayerConnected(playerID); err != nil {
+			log.Fatalf("Error checking if player is connected: %s", err)
+		} else if !ok {
+			log.Fatalf("IsPlayerConnected returned false")
+		}
+
+		if list, err := sdk.Alpha().GetConnectedPlayers(); err != nil {
+			log.Fatalf("Error getting connected player: %s", err)
+		} else if len(list) == 0 {
+			log.Fatalf("No connected players returned")
+		}
+
+		if ok, err := sdk.Alpha().PlayerDisconnect(playerID); err != nil {
+			log.Fatalf("Error registering player as disconnected: %s", err)
+		} else if !ok {
+			log.Fatalf("PlayerDisconnect returned false")
+		}
+
+		if c, err = sdk.Alpha().GetPlayerCount(); err != nil {
```

Comment:
I don't know if it would work, but it would be nice to call this once with the player connected (and verify that result is 1) and then again here to check that it is back to 0. 

Similarly, it would be nice to verify that IsPlayerConnected returns false for a non-existent player and also that it returns false for "1234" after the player disconnects.

I'm sure this is all covered by unit tests, but it would be nice to have it covered by the conformance tests as well.

Reply:
This is definitely covered by the unit tests, but I can't see how we could do this with the current structure of the conformance tests. The conformance tests check that a method has been called once with a specific value, so there is no concept for ordering.

@aLekSer unless you can think of anything I'm not thinking of?

Reply:
That's what I thought. I don't want to block this change, but let's file an issue so we can discuss if it would be worth modifying the conformance test infra.